### PR TITLE
oadm policy reconcile-sccs: update comments and help text

### PR DESCRIPTION
**Pull request by php-coder**
Follow-up improvements to https://github.com/openshift/origin/pull/6714

PTAL @pweil-

### CODE REVIEW

File: pkg/cmd/admin/policy/reconcile_sccs.go [hunk 1]
```diff
@@ -49,9 +50,9 @@ var (
 		This command will inspect the cluster SCCs against the recommended bootstrap SCCs.
 		Any cluster SCC that does not match will be replaced by the recommended SCC.
 		This command will not remove any additional cluster SCCs.  By default, this command
-		will not remove additional users and groups that have been granted access to the SCC and
-		will preserve existing priorities (but will always reconcile unset priorities and the policy
-		definition).
+		will not remove additional users, groups, labels and annotations that have been granted
```
Comment (pweil-) [inline pkg/cmd/admin/policy/reconcile_sccs.go]: labels and annotations are not granted access to the SCC, suggest rewording

Comment (php-coder) [inline pkg/cmd/admin/policy/reconcile_sccs.go]: Good catch! I've fixed it.

**Comment by pweil-** [comment]
[test][merge]

**Comment by php-coder** [comment]
test flake #12072 and bug #13704
